src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.props

```diff
@@ -32,6 +32,59 @@ Copyright (c) .NET Foundation. All rights reserved.
     <UsingNETSdkDefaults>true</UsingNETSdkDefaults>
   </PropertyGroup>
```

Comment:
It's not really possible to do this, because where would we store the cache of the old value?  It would normally go in the intermediate directory, but that may also have changed.

I don't think it's a common scenario to choose a different artifacts / output path within the project folder.  It's more likely that if you are overriding this folder, you will be putting it somewhere further up in the directory tree.  So I don't think it will be too common to hit this.

Reply:
While testing this out for a demo, I did run into issues where after switching to artifacts output format, the intermediate folders from the old `obj` directory were picked up, causing errors.  I had thought they would have been excluded.  I've updated the logic so that when using artifacts output folder the `bin` and `obj` folders in the project are always ignored, and added a test case for switching back and forth between formats.